Sources/XcbeautifyLib/Parser.swift

```diff
@@ -76,18 +76,18 @@ public class Parser {
         innerParser(.compileError, outputType: .error),
         innerParser(.cursor, outputType: .warning),
         innerParser(.fatalError, outputType: .error),
-        innerParser(.ldError, outputType: .error),
         innerParser(.linkerDuplicateSymbolsLocation, outputType: .error),
         innerParser(.linkerDuplicateSymbols, outputType: .error),
         innerParser(.linkerUndefinedSymbolLocation, outputType: .error),
         innerParser(.linkerUndefinedSymbols, outputType: .error),
+        innerParser(.ldError, outputType: .error),
         innerParser(.podsError, outputType: .error),
         innerParser(.symbolReferencedFrom, outputType: .error),
         innerParser(.moduleIncludesError, outputType: .error),
         innerParser(.parallelTestingFailed, outputType: .nonContextualError),
         innerParser(.parallelTestCaseFailed, outputType: .error),
         innerParser(.shellCommand, outputType: .task),
-        innerParser(.undefinedSymbolLocation, outputType: .warning),
+        innerParser(.undefinedSymbolLocation, outputType: .error),
```

Comment:
Hmmm... It's before I started working on this project, but I'm wondering, what's the difference between this and `.linkerUndefinedSymbolLocation`? If both are needed, we should probably distinguish the two with documentation comments.

https://github.com/tuist/xcbeautify/blob/4bbfdc863e175f347674ef0a60ab2048078edea6/Sources/XcbeautifyLib/Parser.swift#L81

Reply:
I'm confused too. Looks like both of these error mesages appear at the stage of linking, but they display in different ways.